source/Octopus.Client/IOctopusClient.cs

```diff
@@ -13,6 +13,8 @@ namespace Octopus.Client
     /// </summary>
     public interface IOctopusClient : IDisposable
     {
+        IOctopusRepository Repository { get; }
```

Comment:
Current `master` does not provide a pre-initialized `Repository` for handy, we add one here. This lines up with `IOctopusAsyncClient`